src/SIL.LCModel/DomainImpl/ScrTxtPara.cs

```diff
@@ -1945,7 +1945,7 @@ protected override void OnContentsChanged(ITsString originalValue, ITsString new
 
 			if ((originalValue == null && newValue != null) ||
 				(originalValue != null && newValue == null) ||
-				(originalValue != null && originalValue.Text != newValue.Text))
+				(originalValue != null && originalValue.Text != newValue!.Text))
```

Comment:
so we only do this when something is not null, or if they're not equal. Could we do this `if (!(originalValue == null && newValue == null) && originalValue?.Text != newValue?.Text)`? is that any better? if both are null then we don't succeed. And then one of the values will be defined so we can compare to a maybe null one.

Reply:
Sure